src/index.js

```diff
@@ -663,11 +703,75 @@ ShowdownEnhancedTooltip.showPokemonTooltip = function showPokemonTooltip(clientP
     if (this.pokemonHasClones(clientPokemon)) {
       text += `(Your opponent has two indistinguishable Pokémon, making it impossible for you to tell which one has which moves/ability/item.) `;
     }
-    text += `</p>`;
   }
+  // Likely Moves
+  const pokemonChaosData = getChaosDataFromLocal(pokemon.speciesForme);
+  if (pokemonChaosData && clientPokemon) {
+    const movesTotal = sumObjectValues(pokemonChaosData.Moves);
+    const likelyMoves = Object.entries(pokemonChaosData.Moves)
+      .sort((a, b) => (a[1] > b[1]) ? -1 : 1)
+      .slice(0, 7) // Only showing the 7 most likely moves
```

Comment:
Only showing the 7 top used moves for now. I could show more, but the menu only has so much space.